### PR TITLE
[P1-L22] initialize fee percentages in Store constructor

### PR DESCRIPTION
**Pull request by mrice32**
Initialize regular and late fees in the Store constructor rather than allowing them to default to 0.

### CODE REVIEW

File: core/contracts/oracle/implementation/Store.sol [hunk 1]
```diff
@@ -43,9 +43,15 @@ contract Store is StoreInterface, Withdrawable, Testable {
...
      */
-    constructor(address _timerAddress) public Testable(_timerAddress) {
+    constructor(
+        FixedPoint.Unsigned memory _fixedOracleFeePerSecondPerPfc,
+        FixedPoint.Unsigned memory _weeklyDelayFeePerSecondPerPfc,
+        address _timerAddress
+    ) public Testable(_timerAddress) {
         _createExclusiveRole(uint(Roles.Owner), uint(Roles.Owner), msg.sender);
         createWithdrawRole(uint(Roles.Withdrawer), uint(Roles.Owner), msg.sender);
+        setFixedOracleFeePerSecondPerPfc(_fixedOracleFeePerSecondPerPfc);
```
Comment (nicholaspai) [inline core/contracts/oracle/implementation/Store.sol:53]: Is there any need to make sure that these are not greater than 100%?

Comment (nicholaspai) [reply core/contracts/oracle/implementation/Store.sol:53]: Similar to how `setFee...` make the checks

File: core/migrations/8_deploy_store.js [hunk 2]
```diff
@@ -9,7 +9,19 @@ module.exports = async function(deployer, network, accounts) {
   const keys = getKeysForNetwork(network, accounts);
   const controllableTiming = enableControllableTiming(network);
 
-  const { contract: store } = await deploy(deployer, network, Store, Timer.address, { from: keys.deployer });
+  // Initialize both fees to 0.
+  const initialFixedOracleFeePerSecondPerPfc = { rawValue: "0" };
```
Comment (nicholaspai) [inline core/migrations/8_deploy_store.js:13]: nit: I noticed that Store.js does not have a unit test for the constructor. Is it worth adding a small one there to check that state variables are set correctly just for peace of mind?

File: core/contracts/oracle/implementation/Store.sol (hunk 1 above)
Comment (mrice32) [reply core/contracts/oracle/implementation/Store.sol:53]: This calls the public function, which does the checks.

File: core/migrations/8_deploy_store.js (hunk 2 above)
Comment (mrice32) [reply core/migrations/8_deploy_store.js:13]: SG, will add a constructor test.

Comment (mrice32) [reply core/migrations/8_deploy_store.js:13]: Done.

**Pull request merged**